### PR TITLE
WUC-6: Storybook

### PR DESCRIPTION
**Pull request by Uzhastin-Nikita**
Installed storybook

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "web-ui-components",
```
Comment (Zimtir) [inline package.json:2]: `@digitable-team/web-ui-components`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Digitable-team/web-ui-components.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (Zimtir) [inline package.json:14]: be sure to discuss with @Adroneleh

File: digitable-storybook/package.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "digitable-storybook",
```
Comment (Zimtir) [inline digitable-storybook/package.json:2]: `@digitable-team/web-ui-components-storybook`

**Review by Zimtir — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,8 @@
-# web-ui-components
\ No newline at end of file
+# web-ui-components
```
Comment (Zimtir) [inline README.md:1]: let's rename to repo name

File: README.md [hunk 5]
```diff
@@ -1 +1,8 @@
...
+
+`cd digitable-storybook`  
+
+`npm start`  
+  
+`npm run storybook`
```
Comment (Zimtir) [inline README.md:7]:
```suggestion

```shell
cd digitable-storybook

npm start
  
npm run storybook
```
```

File: digitable-storybook/.storybook/main.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/preset-create-react-app"
+  ],
+  "framework": "@storybook/react",
+  "core": {
+    "builder": "webpack5"
+  }
+}
```
Comment (Zimtir) [inline digitable-storybook/.storybook/main.js:16]: let's add the empty line at eof

File: digitable-storybook/.storybook/preview.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
```
Comment (Zimtir) [inline digitable-storybook/.storybook/preview.js:9]: let's add the empty line at eof

File: digitable-storybook/README.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (Zimtir) [inline digitable-storybook/README.md:1]: Create GitHub Pages for Storybook

File: digitable-storybook/package.json [hunk 9]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@digitable-team/web-ui-components-storybook",
+  "version": "0.1.0",
+  "private": true,
```
Comment (Zimtir) [inline digitable-storybook/package.json:4]: false

File: digitable-storybook/public/index.html [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (Zimtir) [inline digitable-storybook/public/index.html:12]: Let's change logo file to actual from our web portal

File: digitable-storybook/public/manifest.json [hunk 11]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (Zimtir) [inline digitable-storybook/public/manifest.json:3]: Let's actualize with latest info of project instead default

File: digitable-storybook/src/App.css [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (Zimtir) [inline digitable-storybook/src/App.css:38]: Could you migrate this one to scss?

File: digitable-storybook/src/App.js [hunk 13]
```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
```
Comment (Zimtir) [inline digitable-storybook/src/App.js:2]: Could you add the empty line above?

**Review by Zimtir — CHANGES_REQUESTED** [review]

File: digitable-storybook/README.md (hunk 8 above)
Comment (Uzhastin-Nikita) [reply digitable-storybook/README.md:1]: Created separate issue

File: digitable-storybook/public/index.html (hunk 10 above)
Comment (Uzhastin-Nikita) [reply digitable-storybook/public/index.html:12]: need help

File: package.json (hunk 2 above)
Comment (Zimtir) [reply package.json:14]: @Adroneleh please take a look and resolve

**Comment by Zimtir** [comment]
@Uzhastin-Nikita rebase please

**Comment by Adroneleh** [comment]
@Uzhastin-Nikita , why commits are written in different styles? [link](https://ru.hexlet.io/blog/posts/git-commit-message)

**Comment by Adroneleh** [comment]
@Uzhastin-Nikita , Could you write a description?

File: package.json (hunk 2 above)
Comment (Adroneleh) [reply package.json]: license
@Zimtir , I think we need take GPL license

**Pull request merged**